structure_tests/command_test_v1.go

```diff
@@ -22,6 +22,8 @@ type CommandTestv1 struct {
 	Name           string
 	Setup          [][]string
 	Teardown       [][]string
+	EnvVars        [][]string
```

Comment:
could this be defined as an array of structures with key/value fields? That would make your validation easier.